generators/server/files.js

```diff
@@ -1426,14 +1417,14 @@ const baseServerFiles = {
     {
       condition: generator =>
         (generator.authenticationTypeOauth2 && !generator.applicationTypeMicroservice) ||
-        (!generator.skipUserManagement && generator.databaseTypeSql),
+        (!generator.skipUserManagement && (generator.databaseTypeSql || generator.databaseTypeNeo4j)),
```

Comment:
Maybe we could try to extract the `( || )` part to a new variable, e.g. liquibaseCompatible or something like this?

Reply:
But nothing we must do now. Let's see to make it work and we can do this afterwards.